gradle/docker.gradle

```diff
@@ -29,8 +29,8 @@ ext {
     dockerRetries = project.hasProperty('dockerRetries') ? dockerRetries.toInteger() : 3
     dockerBinary = project.hasProperty('dockerBinary') ? [dockerBinary] : ['docker']
     dockerBuildArg = ['build']
+    dockerFile = project.hasProperty('dockerFile') ? dockerFile : 'Dockerfile'
```

Comment:
No need to change docker.gradle. 
Please replace the current image in https://github.com/apache/incubator-openwhisk-runtime-java/blob/master/core/javaAction/Dockerfile